packages/babel-register/src/node.js

```diff
@@ -128,8 +130,8 @@ export default function (opts?: Object = {}) {
     transformOpts.ignore = [
       new RegExp(
         "^" +
-        escapeRegExp(process.cwd() + path.sep) +
```

Comment:
This regex matched _nested_ `node_modules`but doesn't properly match node_modules directly in the working directory.